crates/wysiwyg/src/dom/to_html.rs

```diff
@@ -25,11 +25,18 @@ where
         buf: &mut S,
         selection_writer: Option<&mut SelectionWriter>,
         state: ToHtmlState,
+        clean: bool,
```

Comment:
Not sure if it makes more sense to add this as an configuration option on the `ToHtml` trait functions (as I have done here) or to add a new trait.